sklearn/utils/extmath.py

```diff
@@ -296,7 +296,7 @@ def randomized_svd(M, n_components, *, n_oversamples=10, n_iter='auto',
         set to `True`, the sign ambiguity is resolved by making the largest
         loadings for each component in the left singular vectors positive.
 
-    random_state : int, RandomState instance or None, default=0
+    random_state : int, RandomState instance or None, default=None
```

Comment:
So here it'd be nice to write that the default is actually `'warn'`, and explain in the docstring what it means and how users should set the value to suppress the warning.